app/controllers/application_controller.rb

```diff
@@ -93,7 +93,9 @@ def check_browser
     return if request_format.xml? || request_format.json?
 
     if current_user && browser_not_modern?
-      logout_keeping_session!
+      logout_killing_session!
```

Comment:
Aha, I see :smile: 
https://github.com/3scale/porta/blob/1506f97e3023964551c28c4a37d45481dd98b59a/app/lib/authenticated_system.rb#L175-L199

Reply:
Why would you keep the cms_token if you want to reset the session?

Reply:
I don't know, it was added inside [a huge PR](https://github.com/3scale/system/commit/1bbc5636e8039a4a724eab44f7580ba1f23513f1) ages ago... without any test :woman_shrugging: